src/blocks/gallery-collage/styles/style.scss

```diff
@@ -142,6 +142,10 @@
 			width: calc(198 / 890 * 100%);
 			z-index: 2;
 
+			&.is-selected {
```

Comment:
@EvanHerman This works great. I think nth-child(4) would also benefit from this. The replace button is visible in english but it might be hidden in other languages. I tested added the same rule on the 4th element and it works pretty well overall. What do you think?

Reply:
I agree that the nth-child(4) should be targeted.